lib/configService.js

```diff
@@ -114,7 +116,23 @@ function getLogger() {
     return logger;
 }
 
+/**
+ * It checks if the configuration file states the use of NGSIv3
+ *
+ * @return     {boolean}  Result of the checking
+ */
+function checkNgsi2() {
```

Comment:
(Similar comment to one in the LWM2M repo PR)

I think to remember the same check at IOTA Library... Do we need double-check (one in the library, one in the agent code)? Could duplicated code be avoided?

Reply:
Fixed in ae72f520f66e348c280580e8baca8306f93ffd76 (it seems so)